apiserver/common/firewall/egressaddresswatcher.go

```diff
@@ -163,7 +165,7 @@ func (w *EgressAddressWatcher) loop() error {
 			if !ok {
 				return w.catacomb.ErrDying()
 			}
-			cfg, err := w.backend.ModelConfig()
+			cfg, err := w.backend.ModelConfig(context.TODO())
```

Comment:
Maybe `context.Background()`?

Reply:
I think that `context.TODO()` is made for usages that you know for sure you want to fix later so explicitly having that method name, `Background()` would better fit usages like tests where you don't care about having the same context passed around (which would break telemetry for example)